packages/ui-kit/src/components/dialogs/confirmation-dialog/confirmation-dialog.tsx

```diff
@@ -56,6 +63,12 @@ const ConfirmationDialog = (props: Props) => (
       dataAttributesSecondaryButton={props.dataAttributesSecondaryButton}
       dataAttributesPrimaryButton={props.dataAttributesPrimaryButton}
       dataAttributesFlatButton={props.dataAttributesFlatButton}
+      displaySecondaryButton={props.displaySecondaryButton}
+      iconLeftSecondaryButton={props.iconLeftSecondaryButton}
+      displayFlatButton={props.displayFlatButton}
+      iconLeftFlatButton={props.iconLeftFlatButton}
+      displayPrimaryButton={props.displayPrimaryButton}
+      iconLeftPrimaryButton={props.iconLeftPrimaryButton}
```

Comment:
I added some options to the confirmation dialog so that we can control all buttons and display icons in them.